jdaviz/core/astrowidgets_api.py

```diff
@@ -178,8 +178,11 @@ def zoom_level(self):
             raise ValueError('Viewer is still loading, try again later')
 
         if hasattr(self, '_get_real_xy'):
-            i_top = get_top_layer_index(self)
-            image = self.layers[i_top].layer
+            if self.state.reference_data is not None:
+                image, i_ref = get_reference_image_data(self.jdaviz_app, self.reference)

```

Comment:
I don't understand this change. Zoom should be based on the visible layer, not the hidden reference data.

Reply:
> Zoom should be based on the visible layer

Why? If they're linked, is this distinction meaningful?

Reply:
Because I want the zoom level of the image under view, not zoom level of the reference data.

Reply:
But the result in example notebook seems correct (when I installed this branch rebased on top of main). Let me think more.

Reply:
Ah, no, it is completely wrong. I forgot to link by WCS when I commented 8 mins ago. When I link by WCS, the zoom_level is wrong. This change needs to be reverted.

Reply:
To give you a sense of what I expect, in the ImvizDitheredExample notebook, a "fit" zoom would be roughly 0.2. A `imviz.default_viewer.zoom_level` of 1 would be like this:

![Screenshot 2024-01-12 125400](https://github.com/bmorris3/jdaviz/assets/2090236/aad20c2d-f8fb-407e-8d89-551352b7ef92)

It should not matter if I am linking by WCS or not.